README.md

```diff
@@ -49,14 +57,16 @@ Suppose you had the following form in your HTML file. Use jQuery to write a sing
   </form>
 ```
 
+var user_input = $('#color').append($'#Submit');
+
 ### Question 7
-This code looks like it works, but when you run it, you see that the `UserApp.add_all_users()` function executes but `console.log` displays `undefined`. What's wrong with the code? 
+This code looks like it works, but when you run it, you see that the `UserApp.add_all_users()` function executes but `console.log` displays `undefined`. What's wrong with the code?
 
 ```
 UserApp.get_all_users = function() {
   $.ajax({
     url: 'http://ig-hacker-news.herokuapp.com/users',
-    type: 'GET',
+    type: 'GET', //<-- *extra comma*
```

Comment:
The comma isn't the problem, actually. The `UserApp.add_all_users(data);` line needs to be inside of the `.done()` callback in order to have access to `data`.